source/server/ssl_context_manager.cc

```diff
@@ -0,0 +1,40 @@
+#include "server/ssl_context_manager.h"
+
+#include "envoy/common/exception.h"
+#include "envoy/registry/registry.h"
+
+namespace Envoy {
+namespace Server {
+
+class SslContextManagerStub final : public Envoy::Ssl::ContextManager {
```

Comment:
nit: I might call this `SslContextManagerNoSslStub` or something like that, but up to you.